readthedocs/metrics/tasks.py

```diff
@@ -0,0 +1,39 @@
+"""
+Specific metric tasks for community.
+
+Override the base metric tasks to add specific ones only required on community.
+"""
+
+# Disable import error because -ext is not available on pylint
+# pylint: disable=import-error
+from readthedocsext.monitoring.metrics.database import RunningBuildsMetric
```

Comment:
Will this not break when the user doesn't have ext installed?

Reply:
I shouldn't break because nobody import this file, but yeah. I don't like it here either. I didn't know where to put them, tho. `metrics/community.py` in -ext sounds better than here. I'll move this code there after the deploy.